upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -283,7 +283,7 @@ func (b *BootstrapChannelBuilder) buildAddons() *channelsapi.Addons {
 
 		{
 			key := "coredns.addons.k8s.io"
-			version := "1.3.0-kops.2"
```

Comment:
Just FYI, in the future you can leave `-kops.0` on the end which just gives us an easy bump for kops specific changes to manifests

Reply:
Oh, nice to know. I will add `-kops.0` then. 👍